airflow_dbt/hooks/dbt_hook.py

```diff
@@ -41,6 +41,8 @@ def __init__(self,
                  dir='.',
                  vars=None,
                  full_refresh=False,
+                 data_test=False,
+                 schema_test=False,
```

Comment:
None of the other variables are suffixed with `_test`, so can we rename them `data` and `schema`?

Reply:
Thanks for taking the time to review Andrew! Changes made in both dbt_hook.py and dbt_operator.py for the variable names and pylint ensures code style adherence.